numpy/polynomial/polynomial.py

```diff
@@ -662,6 +662,10 @@ def polyval(x, c, tensor=True):
     Trailing zeros in the coefficients will be used in the evaluation, so
     they should be avoided if efficiency is a concern.
 
+    This function is preferred over the very similar `numpy.polyval`, which
```

Comment:
See above about `Polynomial`. The other functions in this module are mostly parts out of which Polynomial is constructed. Same for the similar functions for the other polynomial basis and their corresponding objects.